### PR TITLE
feat: add 'status' command to the activate cli

### PR DESCRIPTION
**Pull request by gkc**
**- What I did**
- feat: at_onboarding_cli: add 'status' command to the activate cli to check the status of an atSign
- build: at_onboarding_cli: updated pubspec and changelog for version 1.6.0
- test: fixed a bunch of test failures which were failing in such a way as to not cause the overall test job to fail

**- How I did it**
See commits / diffs

**- How to verify it**
- Run the command e.g. `dart bin/activate_cli.dart -a @no_such_atsign; echo $?`

### CODE REVIEW

File: packages/at_onboarding_cli/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.5.1
```
Comment (cpswan) [inline packages/at_onboarding_cli/CHANGELOG.md]: Should be 1.6.0 as this is definitely new functionality rather than a patch.

**Review by cpswan — CHANGES_REQUESTED** [review]

**Comment by gkc** [comment]
Marking as draft until I understand why the at_onboarding_cli functional tests are failing

File: packages/at_onboarding_cli/CHANGELOG.md (hunk 1 above)
Comment (gkc) [reply packages/at_onboarding_cli/CHANGELOG.md]: @cpswan thank you yes

**Review by xavierchanth — DISMISSED** [review]

File: packages/at_onboarding_cli/CHANGELOG.md (hunk 1 above)
Comment (gkc) [reply packages/at_onboarding_cli/CHANGELOG.md]: have made that change

**Comment by gkc** [comment]
> Marking as draft until I understand why the at_onboarding_cli functional tests are failing

tests failing because of an atServer bug, will be resolved in v 3.0.46 see https://github.com/atsign-foundation/at_server/pull/1958 ... until that has been merged to trunk and VE rebuilt, I will leave this PR as a draft

**Comment by gkc** [comment]
> > Marking as draft until I understand why the at_onboarding_cli functional tests are failing
> 
> tests failing because of an atServer bug, will be resolved in v 3.0.46 see [atsign-foundation/at_server#1958](https://github.com/atsign-foundation/at_server/pull/1958) ... until that has been merged to trunk and VE rebuilt, I will leave this PR as a draft

ready for review

**Review by xavierchanth — APPROVED** [review]

**Review by cpswan — APPROVED** [review]

**Pull request merged**